common/src/main/scala/uk/ac/wellcome/utils/ScanamoUtils.scala

```diff
@@ -17,3 +26,42 @@ object ScanamoUtils {
       .flatMap(_.right.toOption)
   }
 }
+
+object ScanamoQueryStream {
```

Comment:
Since this class is copied together from two classes in Scanamo (it looks like [DynamoResultStream.scala](https://github.com/guardian/scanamo/blob/f358cbfc6ff74c435600ddd2f6e9812aaf81b8e6/src/main/scala/com/gu/scanamo/DynamoResultStream.scala), but I haven't checked in detail), please could we get a comment that:

1. Links back to the original source of this code
2. Highlights anywhere we've deviated from the original